samples/bluetooth/nrf_dm/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -7,3 +7,15 @@
 &ipc0 {
 	zephyr,buffer-size = <1536>;
 };
+
+ /* Enable pin forwarding to network core. The selected pins will be used by
+  * Distance Measurement module for debug purposes.
+  *
+  * Note: Pay attention to assign the same GPIO pins as Kconfig file.
+  */
+&gpio_fwd {
+	dfe-gpio-if {
```

Comment:
Shouldn't this be called somehow related to DM? The dfe is a short from direction finding extension. We use this name in direction finding samples and in the DTM

Reply:
Agree it would be good to call it something related to DM.

Reply:
Fixed.